astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -322,7 +322,8 @@ def test_lstrip_whitespace(parallel, read_basic):
     text = """
      1,  2,   \t3
  A,\t\t B,  C
-  a, b,   c  \n
+  a, b,   c
```

Comment:
@dhomeier - it looks like this change has introduced trailing whitespace and caused pre-commit to file. Can you please just remove this commit?

Reply:
Adding trailing whitespace was probably the point of the original `str + '  \n'` construct.
Any ideas how to implement this in a way that will not let pre-commit fail?

Reply:
I presume that the code that black generated prior to your fix in 9f0450e was fine?
```
    text = """
     1,  2,   \t3
 A,\t\t B,  C
  a, b,   c  \n
"""
```

Reply:
It was passing, but testing a different string (ending in `' a, b,   c  \n\n'`, whereas originally it was `' a, b,   c\n  \n'`).
Feel free to delete the commit, I can't find an option to do it online and will be away from my computer for a few hours. I'd just try the alternative option afterwards, as I don't know what exactly is happening in the pre-commit.